.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+
+### Description
+
```

Comment:
- We should mention links to related issues, and the use of closing keywords.
- OK to repeat title in description

Reply:
I intentionally went with a less-is-more approach on this first iteration. The main point in this is to get the Docs info populated and I don't want to bury that lede.

After this PR I can work on a more holistic template for "how to write a good PR" and then either put it in the template directly, or put it on a wiki page and link to it (to avoid clutter).